src/components/select/multi-select/multi-select.stories.mdx

```diff
@@ -28,12 +27,9 @@ Select multiple options from the drop-down menu. MultiSelect is a component that
 import { MultiSelect, Option } from "carbon-react/lib/components/select";
 ```
 
-## Designer Notes
+Always insert `Option` Components inside the `FilterableSelect`, analogous to the `<select>` and `<option>` HTML Elements.
```

Comment:
```suggestion
Always insert `Option` Components inside the `MultiSelect`, analogous to the `<select>` and `<option>` HTML Elements.
```

Hi @DobroTora, I just noticed we had this typo, please could you amend it and squash the commits again? Sorry to be a pain 😅